README.md

```diff
@@ -30,98 +12,45 @@ Command for running postfix docker container:
 ```
 docker run -it -e MYHOSTNAME=localhost \
     -p 25:10025\
-    -v $(pwd)/tmp/postfix:/var/spool/postfix \
-    -v $(pwd)/tmp/mail:/var/spool/mail postfix
+    -v /var/spool/postfix:/var/spool/postfix \
+    -v /var/spool/mail:/var/spool/mail postfix
+    -d docker.io/modularitycontainers/postfix
 ```
 
 ## How to use the container over standard TLS
 
-Dockerfile for TLS port is Dockerfile.TLS.
-
 Command for running postfix docker container:
 ```
-docker run -it -e DEBUG_MODE=yes -e MYHOSTNAME=localhost \
+docker run -it -e ENABLE_TLS -e DEBUG_MODE=yes -e MYHOSTNAME=localhost \
     -p 25:10025 \
-    postfix
+    postfix -d docker.io/modularitycontainers/postfix
 ```
 
-Environment variable DEBUG_MODE is used for debugging proposes
-from Postfix and dovecot point of views.
+Environment variable DEBUG_MODE is used for debugging purposes
+from postfix point of view.
 
-## How to use the container with IMAP with TLS enabled (dovecot and Postfix)
-
-Dockerfile which uses IMAP is Dockerfile.IMAP.
+## How to use the container with IMAP.
 
 Command for running postfix docker container:
 ```
-docker run -it -e MYHOSTNAME=localhost -e DEBUG_MODE \
-    -p 25:10025 -p 587:10587 -p 143:10143 \
-    postfix
+POSTFIX_CERTS_PATH=/etc/postfix/certs
+docker run -it -e ENABLE_IMAP -e MYHOSTNAME=localhost
+    -e DEBUG_MODE \
+    -p 587:10587 \
+    -v ${POSTFIX_CERTS_PATH}:/etc/postfix/certs \
+    postfix -d docker.io/modularitycontainers/postfix
 ```
+POSTFIX_CERTS_PATH contains certificates used by postfix, like self signed certificate, keys, etc.
+Environment variable DEBUG_MODE is used for debugging purposes from postfix.
 
-Environment variable DEBUG_MODE is used for debugging proposes
-from Postfix and dovecot point of views.
-
-Docker container does NOT share user from HOST system. Therefore
-I prefer to create users in container. This is not yet automated. Will be SOON.
+# How to generate self signed SSL certificate for postfix
 
-Steps are:
+In case, you enable TLS support for postfix, you need to have certificates for postfix service.
 
-```docker exec -n postfix /bin/bash```
-```/file/create_user.sh <user_name>```
+For more information about Postfix TLS support see `http://www.postfix.org/TLS_README.html`
 
-Now the user can be used for mail clients.
+The page `POSTFIX_CERTS_GENERATION.md` will help you with generation self signed certificate used by postfix.
```

Comment:
you can easily link to the page via `[POSTFIX_CERTS_GENERATION.md](/POSTFIX_CERTS_GENERATION.md)`